students/mattmaeda/session07/html_render.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
+# coding=utf-8
+from __future__ import unicode_literals
+
+class Element(object):
+    """ class for rendering HTML element
+    """
+    def __init__(self, content=None, **kwargs):
+        self.tag = "tag"
+        self.elements = []
+        self.kwargs = {}
+
+        if content is not None:
+            self.append(content)
+
+        if kwargs is not None:
+            self.kwargs = kwargs
+
+    def append(self, content):
+        """ Appends content to class attribute elements list
+        """
+        self.elements.append(content)
+
+
+    def render(self, file_out, ind="", depth=1):
+        """ Writes content to StringIO buffer with optional indentation
+        :file_out StringIO: StringIO
```

Comment:
NOTE: file_out is any "file like" object. it MIGHT be a StringIO buffer, but it might jsut as well be a real file.

Which is the entire point of StringIO :-)